target/linux/realtek/image/rtl839x.mk

```diff
@@ -1,5 +1,26 @@
 # SPDX-License-Identifier: GPL-2.0-only
 
+define Device/d-link_dgs-1210-52
+  SOC := rtl8393
+  IMAGE_SIZE := 13824k
+  FACTORY_KERNEL_PART_SIZE := 1572864
+  DEVICE_VENDOR := D-Link
+  FACTORY_IMAGE_SIZE := 12774720
+  DEVICE_MODEL := DGS-1210-52
+  DLINK_KERNEL_PART_SIZE := 1572864
```

Comment:
Since most (or all) of the factory image parameter are shared with the rtl838x images, could you mode this to a Device/d-link_dgs-1210 directive in the main Makefile for the target? The `SOC :=` assignments will have to be moved to the specific devices though.

Reply:
OK. I might have converted it into spaces during copy/paste.

Reply:
I don't think it's worth the hassle to define that for just the shared `SOC`, so no d-link_dgs1210_rtl8382 needed.

Reply:
IMHO a default should only be added if it makes sense for all devices. Defining a `SOC` value that must be overwritten is just waiting for someone to forget, and possibly being confused by the issues that causes.

Reply:
OK. That's what I did. Thanks!